docs/project/contributing.md

```diff
@@ -28,6 +28,24 @@ Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint]
 ## Issues
 First step to be done is access the project you want to contribute to and take a look at the issue tab, maybe there is already an issue open you want to work on, or you can even open one to be discussed.
 
+**Labels** are automatically defined when the issue/PR is opened and are updated if needed by core members.
```

Comment:
@danielsous labels are not automatically defined, just the type of issue label is automatically fixed 

Always an org member tag those issues